pom.xml

```diff
@@ -12,7 +12,7 @@
   <groupId>org.folio</groupId>
   <artifactId>mod-data-export-spring</artifactId>
   <description>Data Export Spring module</description>
-  <version>3.1.0-SNAPSHOT</version>
+  <version>3.2.0-SNAPSHOT</version>
```

Comment:
Why do we need to make this manually? Typically, release plugin makes this.

Reply:
A long-lived release branch with version 3.1.0 (created not from the master) and functionality from the master branch was created urgently for the needs of the team Eureka and LoC. It's like a hybrid of the latest release plus functionality that isn't ready for production. This was done to avoid adding this risky functionality to the current latest releases. Due to the urgency, we deviated a bit from the usual guideline, and to avoid version conflicts, we had to manually increase the version in the master.